stanford/roles/cluster/tasks/rds.yml

```diff
@@ -0,0 +1,27 @@
+- name: Create new subnet group from the subnets
+  rds_subnet_group:
+    state: present
+    name: "{{ COMMON_DEPLOYMENT }}-subnetgroup-{{ CLUSTER_NAME }}"
+    description: "Subnet group for {{ COMMON_DEPLOYMENT }}-subnetgroup-{{ CLUSTER_NAME }}"
+    subnets:
+      - "{{ COMMON_DEPLOYMENT }}-subnet-analyticsdb-1"
+      - "{{ COMMON_DEPLOYMENT }}-subnet-analyticsdb-2"
```

Comment:
If needed, we can do a lookup first with:
https://docs.ansible.com/ansible/latest/modules/ec2_vpc_subnet_facts_module.html#ec2-vpc-subnet-facts-module